js/imageLoad.js

```diff
@@ -65,7 +65,7 @@ function createLogos(logoArray) {
             groupanchor = '';
             alphalinkhtml = '';
         }
-        const html = groupanchor + `<div class='logo'> <div class='logo__holder'> <div class='logo__image'> <img src='logos/${logo.filename}/${logo.filename}.svg' alt='${logo.title}logo'> </div><div class='logo__download'> <div class='logo__download__overlay'> <a href='logos/${logo.filename}/${logo.filename}.svg' download='${logo.title} Logo.svg'> <span class='logo__download__overlay--svg'> Download SVG </span> </a> <a href='logos/${logo.filename}/${logo.filename}.png' download='${logo.title} Logo.png'> <span class='logo__download__overlay--png'> Download PNG </span> </a> </div></div></div><div class='logo__text'> <p class='logo__text--primary'><a href='${logo.url}' title='${logo.urlTitle}' target='_blank' class='logo__text--link'>${logo.title}</a></p><p class='logo__text--secondary'>${(logo.category+"").replace(","," / ")}</p></div></div>`;
+        const html = groupanchor + `<div class='logo'> <div class='logo__holder'> <div class='logo__image'> <img loading="lazy" src='logos/${logo.filename}/${logo.filename}.svg' alt='${logo.title}logo'> </div><div class='logo__download'> <div class='logo__download__overlay'> <a href='logos/${logo.filename}/${logo.filename}.svg' download='${logo.title} Logo.svg'> <span class='logo__download__overlay--svg'> Download SVG </span> </a> <a href='logos/${logo.filename}/${logo.filename}.png' download='${logo.title} Logo.png'> <span class='logo__download__overlay--png'> Download PNG </span> </a> </div></div></div><div class='logo__text'> <p class='logo__text--primary'><a href='${logo.url}' title='${logo.urlTitle}' target='_blank' class='logo__text--link'>${logo.title}</a></p><p class='logo__text--secondary'>${(logo.category+"").replace(","," / ")}</p></div></div>`;
```

Comment:
Neat!

NB: Consistency is key single quote at `<img loading='lazy'` ... not double i.e  `<img loading="lazy"`...